.github/workflows/ci.yml

```diff
@@ -167,6 +167,25 @@ jobs:
           cd build && cmake -DUSE_BUNDLED_DEPS=ON -DCMAKE_BUILD_TYPE=Release -DCREATE_TEST_TARGETS=ON -DMINIMAL_BUILD=ON ..
           cmake --build . --config Release && make run-unit-tests || libsinsp\test\Release\unit-test-libsinsp.exe
 
+  build-shared-libs-others-amd64:
+    name: build-shared-libs-others-amd64 😨
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Libs ⤵️
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build and test 🏗️🧪
+        run: |
+          mkdir -p build
+          cd build && cmake -DBUILD_SHARED_LIBS=True -DUSE_BUNDLED_DEPS=ON -DCMAKE_BUILD_TYPE=Release -DCREATE_TEST_TARGETS=OFF -DMINIMAL_BUILD=ON ..
+          cmake --build . --config Release && make run-unit-tests || libsinsp\test\Release\unit-test-libsinsp.exe
```

Comment:
```
&& make run-unit-tests || libsinsp\test\Release\unit-test-libsinsp.exe
``` 
Since we are not creating any test target, we cannot run tests.
I'd follow what `build-shared-libs-linux-amd64:` is doing, ie: building sinsp-example and try to run it to check if runtime linking is working fine:
```
- name: Test sinsp-example build with pkg-config
        run: |
          cd userspace/libsinsp/examples
          export PKG_CONFIG_PATH=/tmp/libs-test/lib/pkgconfig
          g++ -o sinsp-example test.cpp util.cpp $(pkg-config --cflags --libs libsinsp)
          
      - name: Test sinsp-example runtime linker
        run: |
          cd userspace/libsinsp/examples
          export LD_LIBRARY_PATH=/tmp/libs-test/lib/falcosecurity
./sinsp-example -h
```

Reply:
Done. The changes ended up being specific to macOS, so I changed the job name to "build-shared-libs-macos-amd64". I also needed to add "--std=c++17" to the test compile flags. I'm not sure if we should add that to libsinsp.pc.